docker-compose.frontend.yml

```diff
@@ -33,6 +32,15 @@ services:
       # Allow access to KoboCAT media uploads within KPI
       - ./.vols/kobocat_media_uploads:/srv/src/kobocat/media
     restart: unless-stopped
+    # NOTE: DO WE NEED THIS ATTRIBUTE IN THE ANCHOR FOR ALL WORKER CONTAINERS? 
+    # PROPABLY, ONLY KPI SERVICE NEEDS TO CONNECT TO kobo-fe-network!!
+    networks:
+      kobo-fe-network:
+
+services:
+  kpi:
+    <<: *django
+    hostname: kpi
```

Comment:
I realize in the PR description you wrote:

> in order to minimize the diff, I did not add hostname: service_name to each worker service. But it would be a good idea to do so for clarity and explicitness.

I'd argue that we should do the opposite, and remove this lone `hostname` since the well-documented and understood Docker Compose behavior is for the hostname to automatically be set to the service name:

> By default, any service can reach any other service at that service's name. In the following example, db is reachable from web at the hostnames db and database:
> …

https://docs.docker.com/compose/how-tos/networking/#link-containers